src/scriptdev2/scripts/kalimdor/ruins_of_ahnqiraj/boss_ossirian.cpp

```diff
@@ -82,9 +82,11 @@ struct boss_ossirianAI : public ScriptedAI
     uint32 m_uiCycloneTimer;
     uint32 m_uiStompTimer;
     uint32 m_uiSilenceTimer;
+    uint32 m_uiSpeedUpTimer;
     uint8 m_uiCrystalPosition;
 
     bool m_bSaidIntro;
+    bool m_bSpeedUp;
```

Comment:
I think you can skip this boolean variable by setting `m_uiSpeedUpTimer` to 0 once the timer expired and checking if it is zero or not, instead.